docs/manage/users-roles/roles/rbac-for-indexes.md

```diff
@@ -63,6 +66,14 @@ An advanced filter allows access only to the logs that match the search filter.
    * **Security data filter**. This filter applies on all logs in [Cloud SIEM security indexes](/docs/cse/records-signals-entities-insights/search-cse-records-in-sumo#partitions-for-cse-records).
 1. Enter search criteria in the box provided. For examples, see [Understanding search filters](/docs/manage/users-roles/roles/construct-search-filter-for-role#understanding-search-filters).<br/><img src={useBaseUrl('img/users-roles/advanced-filter.png')} alt="Advanced filter" style={{border: '1px solid black'}} width="400"/>
 
+### Advanced filter examples
+
+Following are examples for advanced filtering:
+* Let’s say you want to deny access to all logs that contain “error” in log analytics, and contain “malicious=high” in security logs. Select **Log Analytics data filter** and add `!error` to the filter, and then select **Security data filter** and add `!malicious=high` to the filter. 
```

Comment:
should these be in monospace, too?

```suggestion
* Let’s say you want to deny access to all logs that contain `error` in log analytics, and contain `malicious=high` in security logs. Select **Log Analytics data filter** and add `!error` to the filter, and then select **Security data filter** and add `!malicious=high` to the filter. 
```

Reply:
Yes, @kimsauce. That's a good suggestion.